github-repos/src/redux/actions.js

```diff
@@ -0,0 +1,11 @@
+
+
+export const fetchRepos = username => (dispatch) => {
+  dispatch({ type: 'LOADING' });
+  fetch(`https://api.github.com/users/${username}/repos`)
+    .then((response) => {
+      dispatch({ type: 'SUCCESS' });
+      return response.json();
+    })
+    .then(data => dispatch({ type: 'FETCH_REPOS', payload: data.map(repo => repo.name) }));
```

Comment:
Here the then callback is taking just the response ( means only one parameter), you should preferably use response without braces 
 Prefer this
 `.then(response => {`
instead of `this((response) => {`